src/libstd/lib.rs

```diff
@@ -400,6 +400,15 @@ pub mod __rand {
     pub use rand::{thread_rng, ThreadRng, Rng};
 }
 
+// Rust runtime's startup objects depend on these symbols, so they must be public.
+// Since sys_common isn't public, we have to re-export them here explicitly.
+#[doc(hidden)]
+#[unstable(feature = "eh_frame_registry", issue = "0")]
+#[cfg(all(target_os="windows", target_arch = "x86", target_env="gnu"))]
+pub mod __frame_registry {
+    pub use sys_common::unwind::imp::eh_frame_registry::*;
+}
```

Comment:
Could this be moved into `src/libstd/rt.rs`? That's got other public symbols like some unwinding ones, and it's already auto-unstable as well.